pyproject.toml

```diff
@@ -1,12 +1,12 @@
 [tool.poetry]
 name = "drsclient"
-version = "0.1.4"
+version = "0.1.5"
 description = "GA4GH DRS Client"
 authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.6,<=3.9"
```

Comment:
could you update https://github.com/uc-cdis/drsclient/blob/master/.travis.yml to test 3.9